src/shared-components/accordion/index.tsx

```diff
@@ -32,131 +32,97 @@ type AccordionProps = {
+      setContentHeight(nextHeight);
     }
-  }
+  });
 
-  handleKeyDown = (event: React.KeyboardEvent): void => {
-    const { onClick, disabled } = this.props;
+  const handleKeyDown = (event: React.KeyboardEvent): void => {
     if (!disabled && event.key === 'Enter') {
       onClick(event);
     }
   };
 
-  render() {
-    const { contentHeight } = this.state;
-    const {
-      borderRadius,
-      title,
-      isOpen,
-      onClick,
-      children,
-      noBorder,
-      disabled,
-      rightAlignArrow,
-    } = this.props;
+  return (
+    <AccordionBox isOpen={isOpen} noBorder={!!noBorder} disabled={!!disabled}>
+      <TitleWrapper
+        onClick={(event): void => {
+          if (!disabled) {
+            onClick(event);
+          }
+        }}
+        borderRadius={borderRadius}
+        onKeyDown={handleKeyDown}
+        disabled={!!disabled}
+        role="button"
+        isOpen={isOpen}
+        tabIndex={disabled ? -1 : 0}
+        aria-disabled={!!disabled}
+        aria-expanded={isOpen}
+      >
+        <Truncate>{title}</Truncate>
+        <ArrowWrapper rightAlign={!!rightAlignArrow}>
+          <ChevronIcon
+            rotate={isOpen ? 90 : 0}
+            width={16}
+            height={16}
+            fill={COLORS.primary}
+          />
+        </ArrowWrapper>
+      </TitleWrapper>
+      <ExpansionWrapper
+        contentHeight={contentHeight}
+        aria-disabled={!!disabled}
+        aria-hidden={!isOpen}
+      >
+        <div ref={contentRef}>{children}</div>
+      </ExpansionWrapper>
+    </AccordionBox>
+  );
+};
 
-    return (
-      <AccordionBox isOpen={isOpen} noBorder={!!noBorder} disabled={!!disabled}>
-        <TitleWrapper
-          onClick={(event): void => {
-            if (!disabled) {
-              onClick(event);
-            }
-          }}
-          borderRadius={borderRadius}
-          onKeyDown={this.handleKeyDown}
-          disabled={!!disabled}
-          role="button"
-          isOpen={isOpen}
-          tabIndex={disabled ? -1 : 0}
-          aria-disabled={!!disabled}
-          aria-expanded={isOpen}
-        >
-          <Truncate>{title}</Truncate>
-          <ArrowWrapper rightAlign={!!rightAlignArrow}>
-            <ChevronIcon
-              rotate={isOpen ? 90 : 0}
-              width={16}
-              height={16}
-              fill={COLORS.purple}
-            />
-          </ArrowWrapper>
-        </TitleWrapper>
-        <ExpansionWrapper
-          contentHeight={contentHeight}
-          aria-disabled={!!disabled}
-          aria-hidden={!isOpen}
-        >
-          <div ref={this.contentRef}>{children}</div>
-        </ExpansionWrapper>
-      </AccordionBox>
-    );
-  }
-}
+Accordion.Container = Container;
+
+Accordion.Content = Content;
+
+Accordion.Thumbnails = Thumbnails;
+
+Accordion.Truncate = Truncate;
+
+Accordion.propTypes = {
+  /** Sets the border-radius of Accordion.Container, AccordionBox, and TitleWrapper */
```

Comment:
These were originally set on `propTypes` because it was required for class components to populate the Docs page. The storybook for this PR has lost these comments (https://curology-radiance-pr-451.herokuapp.com/?path=/docs/components-accordion), so we'll have to move the JSDoc comments to the Type definition instead of `propTypes`.

Reply:
Oh these needs to go to the AccordionProps yes

Reply:
done